src/main/java/apoc/periodic/Periodic.java

```diff
@@ -227,7 +235,9 @@ public static JobInfo schedule(String name, Runnable task, long delay, long repe
             @Name("cypherIterate") String cypherIterate,
             @Name("cypherAction") String cypherAction,
             @Name("batchSize") long batchSize) {
-
+        validateQuery(cypherLoop);
```

Comment:
for the places where we have several checks we should collect the errors and return them together.
so perhaps a varags parameter and iterate over them and collecting the errors per statement

Reply:
done